charts/partials/templates/_proxy.tpl

```diff
@@ -167,8 +167,7 @@ lifecycle:
 {{- if .Values.proxy.await }}
   postStart:
     exec:
-      command:
-      - /usr/lib/linkerd/linkerd-await
+      command: ["/usr/lib/linkerd/linkerd-await", "-t", "2m"]
```

Comment:
nit:

```suggestion
      command:
        - /usr/lib/linkerd/linkerd-await
        - --timeout=2m
```

* the long-form flag is clearer to me
* we can combine the flag and value in one item so that if it gets expanded they stay together
* using the expanded list form makes future changes simpler (i.e. when more flags are required)